docs/definition_of_ready.md

```diff
@@ -0,0 +1,12 @@
+# Definition of ready
```

Comment:
Wat in DOR staat lijkt punten voor de developers. Gelden deze ook voor de ontwerpers van de OAS?

Reply:
Volgens mij gelden deze punten ook voor het design van de API.